### PR TITLE
feat(FN-1615): add generic for express render options

### PR DESCRIPTION
**Pull request by JoshBinns2000**
## Introduction :pencil2:
It would be nice to have a way of defining the type we want to use for the express `res.render` method.

## Resolution :heavy_check_mark:
- Adds a new declaration file to merge the declaration for the express `core.Response.render` function
- Updates `res.render` in the controller for `getUtilisationReportReconciliationForReport` in TFM-UI

### CODE REVIEW

File: trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/index.ts [hunk 1]
```diff
@@ -9,9 +9,14 @@ export const getUtilisationReportReconciliationByReportId = async (req: Request,
   const { reportId } = req.params;
 
   try {
-    const utilisationReportReconciliationDetails = await api.getUtilisationReportReconciliationDetailsById(reportId, userToken);
+    const utilisationReportReconciliationDetails = await api.getUtilisationReportReconciliationDetailsById(
+      reportId,
+      userToken,
```
Comment (abhi-markan) [inline trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/index.ts:14]: Various GitHub alerts on type any.

File: trade-finance-manager-ui/server/types/express-serve-static-core.d.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (abhi-markan) [inline trade-finance-manager-ui/server/types/express-serve-static-core.d.ts:1]: Please remove inline rule override.

**Review by abhi-markan — CHANGES_REQUESTED** [review]
If this approach is must then I would prefer being applied across other UI microservices for consistency.

**Comment by abhi-markan** [comment]
@JoshBinns2000 is this PR still required or can we close this?

**Comment by JoshBinns2000** [comment]
The pattern of adding a `render<View>` function at the controller level has been added in preference to this typescript-first approach

**Pull request closed without merging**